code/modules/mob/mob.dm

```diff
@@ -124,6 +124,65 @@
 	if(bound_overlay)
 		bound_overlay.visible_message(message, blind_message, range)
 
+/**
+ * Show a message to all mobs and objects in sight of this one or `causer`
+ * This should be used for messages where two mobs interact - healing, injections, fighting, and so on
+ * message is the message output to anyone who can see, e.g. "[causer] does something to [src]!"
+ * self_message (optional) is what the src mob sees, e.g. "[causer] does something to you!"
+ * causer_message is what the causer mob sees, e.g. "You do something to [src]!"
+ * blind_message (optional) is what blind people will hear, e.g. "You hear something!"
+ * blind_self_message (optional) is what the source mob will hear/feel if blind, e.g. "You feel something done to you!"
```

Comment:
Since this is direct interaction between two people, I can't actually foresee a case in which audible messages would be required here. Someone being shot should always still feel being shot, even if they're blind and deaf, thanks to `blind_self_message`; the causer will always know what happened due to being the one to cause it, and observers will see or hear the message due to `blind_message` existing.